config/analytics_blocklist.yml

```diff
@@ -0,0 +1,505 @@
+  - secondary_contact_email
+  - opened_on
+  - closed_on
+  - created_at
+  - updated_at
+  - administrative_district_name
+  - local_authority_code
+  - local_authority_name
+  - establishment_number
+  - phase_name
+  - section_41_approved
+  - type_name
+  - ukprn
+  - website
+  - induction_eligibility
+  - in_england
+  :users:
+  - id
+  - name
+  - email
+  - otp_secret
+  - otp_verified_at
+  - created_at
+  - updated_at
+  - full_name
+  - login_token
+  - login_token_valid_until
+  - remember_created_at
+  - last_sign_in_at
+  - current_sign_in_at
+  - current_sign_in_ip
+  - last_sign_in_ip
+  - sign_in_count
+  - discarded_at
+  - get_an_identity_id
+  - archived_email
+  - archived_at
+  :training_periods:
+  - id
+  - provider_partnership_id
+  - started_on
+  - finished_on
+  - created_at
+  - updated_at
+  - ect_at_school_period_id
+  - mentor_at_school_period_id
+  - range
+  - legacy_start_id
+  - legacy_end_id
+  :teacher_migration_failures:
+  - id
+  - teacher_id
+  - message
+  - migration_item_id
+  - migration_item_type
+  - created_at
+  - updated_at
+  :provider_partnerships:
+  - id
+  - academic_year_id
+  - lead_provider_id
+  - delivery_partner_id
+  - created_at
+  - updated_at
+  :pending_induction_submissions:
+  - id
+  - appropriate_body_id
+  - establishment_id
+  - trn
+  - trs_first_name
+  - trs_last_name
+  - date_of_birth
+  - trs_induction_status
+  - induction_programme
+  - started_on
+  - finished_on
+  - number_of_terms
+  - created_at
+  - updated_at
+  - confirmed_at
+  - trs_email_address
+  - trs_alerts
+  - trs_induction_start_date
+  - trs_induction_status_description
+  - trs_qts_status_description
+  - trs_initial_teacher_training_end_date
+  - trs_initial_teacher_training_provider_name
+  - outcome
+  - trs_qts_awarded_on
+  - confirmed
+  :migration_failures:
+  - id
+  - data_migration_id
+  - item
+  - failure_message
+  - created_at
+  - updated_at
+  - parent_id
+  - parent_type
+  :mentorship_periods:
+  - id
+  - ect_at_school_period_id
+  - mentor_at_school_period_id
+  - started_on
+  - finished_on
+  - created_at
+  - updated_at
+  - range
+  - legacy_start_id
+  - legacy_end_id
+  :mentor_at_school_periods:
+  - id
+  - school_id
+  - teacher_id
+  - started_on
+  - finished_on
+  - created_at
+  - updated_at
+  - range
+  - legacy_start_id
+  - legacy_end_id
+  - email
+  :lead_providers:
+  - id
+  - name
+  - created_at
+  - updated_at
+  - cpd_lead_provider_id
+  - vat_chargeable
+  :events:
+  - id
+  - heading
+  - body
+  - event_type
+  - happened_at
+  - teacher_id
+  - appropriate_body_id
+  - induction_period_id
+  - induction_extension_id
+  - school_id
+  - ect_at_school_period_id
+  - mentor_at_school_period_id
+  - training_period_id
+  - mentorship_period_id
+  - provider_partnership_id
+  - lead_provider_id
+  - delivery_partner_id
+  - user_id
+  - author_type
+  - author_id
+  - author_name
+  - author_email
+  - created_at
+  - updated_at
```

Comment:
Sorry, can we move this to `config/analytics.yml` - events will be useful for the data team, but `heading` and `body` to the `config/analytics_hidden_pii.yml`?